### DISCUSSION TITLE
SBFspot standalone Proxmox VM Image

### DISCUSSION THREAD

**Discussion by alfbenn** [SBFspot/SBFspot#607] [Q&A]
Hi all,

thanks for this great tool, it's running for years now on a raspberry pi.
Now I want to set up a new linux (debain or ubuntu) on a proxmox server on a HP thinclient T 630.
Is there maybe any source vm image that I can download it, because I'm a fully linux beginner.

Thanks for your ideas.

Greetings from Northern Germany
Alf

**Comment by jofland** [comment]
Bei mir läuft das in einem docker Container. Vielleicht ist es für dich sinnvoll, eine proxmox vm für docker zu nutzen und darin unter anderen sbfspot zu hosten.

**Comment by alfbenn** [comment]
Hi,
Danke für den Tipp, inzwischen läuft SBFSpot in einer Proxmox VM, soweit auch fehlerfrei. Einzig nach Reboot des Proxmox Hosts habe ich Probleme mit den USB Devices

VG Alf
